src/package-hoister.js

```diff
@@ -428,7 +427,7 @@ export default class PackageHoister {
     // update to the new key
     if (oldKey === newKey) {
       info.addHistory("Didn't hoist - conflicts above");
-      this.setKey(info, oldKey, parts);
+      this.setKey(info, oldKey, rawParts);
```

Comment:
It is and it corrects the behavior actually since the intention was to re add the removed one to cause no changes if there are conflicts. Right?